app/assets/stylesheets/content/_flash_messages.sass

```diff
@@ -42,6 +42,7 @@
 
 .flash
   display: none
+  padding: 15px 5px
```

Comment:
> whole layout is not relative-ready

But we could start here.

But as I said, it is just an opinion. We can do the work later.